protos/perfetto/config/priority_boost/BUILD.gn

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2025 The Android Open Source Project
```

Comment:
I don't see the reason of creating a dedicated subdirectory to just host one single .proto file.
please move this one directory up.
You can still have a dedicated target if you really need it